react/components/molecules/tabs-text/tabs-text.js

```diff
@@ -45,13 +46,37 @@ export class TabsText extends PureComponent {
             animatedBarOffset: undefined
         };
         this.tabLayouts = {};
+        this.XScroll = 0;
+        this.overflows = false;
     }
 
     onTabPress = tabSelectedIndex => {
         this.setState({ tabSelected: tabSelectedIndex }, () => {
             this._updateBar();
             this.props.onTabChange(this.state.tabSelected);
         });
+        const deviceWidth = Dimensions.get("window").width;
+        const tabLayout = this.tabLayouts[tabSelectedIndex];
+        const overFlowRight = tabLayout.x + tabLayout.width > deviceWidth;
+        const overFlowLeft = this.XScroll > tabLayout.x;

```

Comment:
Overflow is a single word
```suggestion
        const overflowRight = tabLayout.x + tabLayout.width > deviceWidth;
        const overflowLeft = this.XScroll > tabLayout.x;
```